docs/guides/view_engines/handlebars.md

```diff
@@ -20,6 +20,20 @@ To enable [**handlebars**](https://handlebarsjs.com) support make use of the `vi
 
 Handlebars is imported dynamically, so make sure to install it (`npm i hbs`). Otherwise `nestjs-i18n` can't register the helper function.
 
+## Fastify
+
+To use handlebars with fastify, `handlebars` is used instead of `hbs`. Make sure to install it (`npm i handlebars`).
```

Comment:
You have already installed it as a dependency in `package.json`, so need to add this make sure to install at readme.